src/locales/de-DE.json

```diff
@@ -21,7 +23,8 @@
   "errors": {
     "KEYCLOAK_CLIENT_ERROR": "Die Anmeldung ist momentan nicht möglich.",
     "UNSPECIFIED_ERROR": "Es ist ein Fehler aufgetreten.",
-    "USER_AUTHENTICATION_FAILED_ERROR": "Ihr Benutzername oder Ihr Passwort war falsch. Versuchen Sie es erneut oder wenden Sie sich an Ihre schulischen Administratorinnen und Administratoren."
+    "USER_AUTHENTICATION_FAILED_ERROR": "Ihr Benutzername oder Ihr Passwort war falsch. Versuchen Sie es erneut oder wenden Sie sich an Ihre schulischen Administratorinnen und Administratoren.",
+    "USER_DATA_LOADING_ERROR": "Fehler beim Laden des Benutzers"
```

Comment:
Die anderen sind nur groß geschrieben, weil es entsprechende Konstanten in Code gibt. Ist für diese Meldung nicht nötig.
Die Texte sind merkwürdig aufgeteilt. Warum ist der Titel bei den Errors und der Text im Abschnitt admin.user? Bitte nebeneinander legen.

Reply:
Meine Logik war, dass die Titles unter "errors" alle Fehlermeldungen kategorisieren sollten, , weil sie in verschiedenen Kontexten verwendet werden können, nicht nur im Rahmen von Adminbereich. Während "admin.user" für Verwaltungsmaßnahmen im Zusammenhang mit der Benutzerverwaltung gedacht war.